app/scripts/lib/oauth-client.js

```diff
@@ -39,7 +47,19 @@ define(function (require, exports, module) {
     },
 
     // params = { assertion, client_id, scope }
-    getToken: function (params) {
+    /**
+     *
+     * @param {Object} params
+     * @param {String} params.assertion
+     * @param {String} params.client_id
+     * @param {String} params.scope
+     * @returns {Promise}
+     */
+    getToken: function (params = {}) {
+      // set authorization TTL to 5 minutes.
+      // Docs: github.com/mozilla/fxa-oauth-server/blob/master/docs/api.md#post-v1authorization
+      // Issue: #3982
+      params.ttl = '300';
```

Comment:
@vladikoff Thoughts on making this configurable via [params](https://github.com/mozilla/fxa-content-server/blob/master/app/scripts/models/account.js#L216)? Maybe overkill?